lib/KettleServer.js

```diff
@@ -0,0 +1,303 @@
+/*
+Kettle Server component, corresponding directly with an express/node HTTP server
+
+Copyright 2012-2013 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/fluid-project/kettle/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    http = require("http"),
+    express = require("express"),
+    kettle = fluid.registerNamespace("kettle");
+
+fluid.defaults("kettle.server", {
+    gradeNames: ["fluid.component"],
+    mergePolicy: {
+        rootMiddleware: "noexpand"
+    },
+    rootMiddleware: { // free hash of middleware namespaces to components
+        urlencoded: {
+            middleware: "{middlewareHolder}.urlencoded",
+            priority: "before:json"
```

Comment:
This is a nice improvement over the previous `kettle.use.bodyParser` approach, as it is clear how to arrange middleware in relation to each other to avoid the kinds of problems we saw with [`express-couchUser`](https://github.com/twilson63/express-couchUser) and the `bodyParser` middleware.